### ISSUE TITLE
Suggested Return Command to Main Server (RedirectPlus)

### ISSUE THREAD

**Issue by AxelJoue** [MartenM/RedirectPlus#14]
**Hello,**

Would it be possible for the commands of the Redirectplus plugin that we configure to return to the lobby to be suggested when we are ready to execute the command ?

Currently when writing for example for my "/lobby" case, the command is displayed as if it did not exist :
**IMG :** https://media.discordapp.net/attachments/426003895333158912/723519761983733770/unknown.png

The "aliases" commands should be suggested when they are configured in the config, like this for example :
**IMG :** https://media.discordapp.net/attachments/426003895333158912/723519673957613619/unknown.png

**Config example :**

```
lobby:
# If kicked from this server group, no redirects will happen.
bottom-kick: false
# Spread redirected people over the servers.
spread: true
# Spread mode, if spread is enabled, use the following spread mode (will overwrite global if used).
spread-mode: PROGRESSIVE
# The minimal amount of players that are on a server before the next server gets selected. (only used for PROGRESSIVE spread)
progressive-minimal: 20
# Parent group - If none of the servers are available, the parent group will be used.
# Set to none if none. If the parent server group is needed the player will be kicked instead.
parent-group: secours
servers:
- Hub01
servers-regex: 'none'
# Servers that are connected to this group, but they won't be redirected to:
 connected: {}
 connected-regex: 'none'
# Aliases can be used to redirect people to the server group when they use a certain command (Eg: lobby, hub, survival, towny, etc)
aliases:
- lobby
- hub
- retour
- leave
```

> aliases:
> - lobby
> - hub
> - retour
> - leave

**Thank you !**

**Cordially,**
**Axel_Joue**

(Excuse me for my English, I am French)

**Comment by MartenM** [comment]
I actually looked into this and due to how things are currently done it's not implemented that easily. Will do some more research on it.

**Comment by MartenM** [comment]
Implemented in f39a2dee0774b3bbdbfab2861286e2a5b96ab95f included in version `2.12.0`.

🎉
